rpc/js/src/dial.ts

```diff
@@ -138,13 +156,13 @@ async function makeAuthenticatedTransportFactory(
         creds.setPayload(opts.credentials?.payload!);
         request.setCredentials(creds);
 
-        let done = new Promise<grpc.Metadata>((resolve, reject) => {
+        const done = new Promise<grpc.Metadata>((resolve, reject) => {
```

Comment:
can you explain this change? it seems like we are changing this variable later on